traffic_portal/test/end_to_end/users/pageData.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+module.exports = function(){
+    this.username=element(by.name('uName'));
```

Comment:
this doesn't look right. on line 61 of table.users.tpl.html you have:

```
 <td name="username" data-search="^{{::u.username}}$">{{::u.username}}</td>
```

not sure where `uName` came from

Reply:
this might explain your broken TP tests

Reply:
oh nevermind, i think i'm gettng the table and form confused.